libraries/psibase/common/include/psibase/version.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#define PSINODE_VERSION_MAJOR 0
```

Comment:
Since this is a single project-wide version number, it should probably be called `PSIBASE` rather than `PSINODE`